src/button/styled-components.js

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import {styled} from '../styles';
+import {KIND, SIZE, SHAPE} from './constants';
+
+export const BaseButton = styled(
+  'button',
+  ({$theme, $size, $kind, $shape}) => ({
+    ...$theme.typography[$size === SIZE.compact ? 'font200' : 'font300'],
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    border: 'none',
+    borderRadius: $shape === SHAPE.round ? '50%' : $theme.borders.radius200,
+    textDecoration: 'none',
+    outline: 'none',
```

Comment:
Forgot to mention this – if we're removing outline, can we add another style for focus? When tabbing through buttons right now I can't tell which one is focused.

Reply:
Thanks for bringing this up. It's most likely an a11y concern that I need to dig further into. Going to clump it with the a11y todos here: https://github.com/uber-web/baseui/issues/207